doc/langref.html.in

```diff
@@ -8964,8 +8966,12 @@ all your base are belong to us</code></pre>
       {#header_close#}
       {#header_close#}
       {#header_open|WebAssembly#}
+      <p>Zig supports building for WebAssembly out of the box. There is also a specialized {#syntax#}std.heap.wasm_allocator{#endsyntax#}
+      memory allocator for WebAssembly environments.</p>
       {#header_open|Freestanding#}
-      {#code_begin|lib|wasm#}
+      <p>For embedded environments like the web browser and nodejs, build as a library using the freestanding OS target.
```

Comment:
'embedded' isn't a great word to use here. Too easy to confuse with the idea of "embedded devices"

Reply:
It’s the language that the spec uses https://webassembly.github.io/spec/core/appendix/embedding.html